proto/stride/auction/auction_pool.proto

```diff
@@ -0,0 +1,104 @@
+syntax = "proto3";
+
+package stride.auction;
+
+import "google/protobuf/any.proto";
+import "stride/auction/bid.proto";
+import "stride/auction/params.proto";
+import "stride/auction/properties.proto";
+
+
+option go_package = "github.com/Stride-Labs/stride/v5/x/auction/types";
+
+/* ****************************************************************************
+An AuctionPool is a combination of an address which holds assets and rules 
+that any auctions for those assets would be restricted to follow.  It is like
+a potential auction defining the bounds of what is allowed.
+
+In contrast, an Auction is a specific instance which is either running now
+or completed in the past, for a specific amount, with specific param settings. 
+Auction contains fields which are common across all different auction types
+
+But depending on the algorithm which the auction is running, there are some
+algorithm specific parameters.
+
+
+RedemptionRate is important to what a bid means -- if it is exactly 0, then
+the price is a ratio the bidder is willing to pay between the auctionDenom
+and the bidDenom.  If RR is any other number then the RR represents the fixed
+rate which auctionDenom trades against bidDenom and the price is an additional
+marginal price the bidder is willing to pay in order to get to exchange at this
+known fixed rate.  If the order is partially filled, this additional price will
+proportionally also be charged.
+
+The marginal model is "I will pay Price to convert Volume A to B at fixed RR"
+The ratio model is "I will convert Volume A into B at Price ratio of A to B"
+
+**************************************************************************** */
+
+enum AuctionState {
+  COMPLETE = 0;
+  RUNNING = 1;
+  REVEAL = 2;
+  PAYOUT = 3;
+}
+
+// An Auction is a specific auction for a specific target asset and supply 
+// even if the supply in the address of the pool changes during that time
+// Depending on the algorithm chosen there are different specific params
+
+// English style auction specific parameters
+// Starts low with increasing bids, auction ends if no new bids occur 
+// for endDuration blocks in a row and goes to current winner
+// When a higher bid comes in, the lastBlock is set now + endDuration
+message AscendingAuction {
+  AuctionState status = 1;
+  AscendingAuctionProperties auctionProperties = 2;
```

Comment:
The common practice for defining message fields is to use snake_case, not camelCase.
Please convert all camelCase fields to snake_case.

Reply:
So won't this make the type private to the module?  Or will the proto generation step still generate it with different capitalization so that it can be used across different packages?